fulp_modules/features/antagonists/bloodsuckers/code/antagonists/bloodsucker_datum.dm

```diff
@@ -24,6 +24,10 @@
 	var/bloodsucker_title
 	///Used for assigning your reputation
 	var/bloodsucker_reputation
+	///HUD color
+	var/vassal_color
+	///antag_hud_name is dynamic because of colors. Update THIS, not antag_hud_name.
+	var/base_hud_name = "bloodsucker"
```

Comment:
This could probably just be a define

Reply:
base_hud_name gets changed when masquarade gets broken.